packages/ppx_rapper/ppx_rapper.3.1.0/opam

```diff
@@ -31,7 +31,7 @@ build: [
 ]
 dev-repo: "git+https://github.com/roddyyaga/ppx_rapper.git"
 url {
-  src: "https://github.com/roddyyaga/ppx_rapper/archive/3.1.0.tar.gz"
+  src: "https://raw.githubusercontent.com/ocaml/opam-source-archives/HEAD/ppx_rapper-3.1.0.tar.gz"
```

Comment:
The intention is to use links that don't hardcode the name of the default branch, to avoid the same churn that happened with master -> main...